pyof/v0x04/common/flow_match.py

```diff
@@ -339,7 +339,8 @@ def __init__(self, match_type=None, oxm_match_fields=None):
         """
         super().__init__()
         self.match_type = match_type
-        self.oxm_match_fields = oxm_match_fields
+        self.oxm_match_fields = (OxmMatchFields() if oxm_match_fields is None
+                                 else oxm_match_fields)
```

Comment:
Quite creative style :). It's much more common to see the below when it doesn't fit in 80 columns:
```python
if oxm_match_fields is None:
    oxm_match_fields = OxmMatchFields()
self.oxm_match_fields = oxm_match_fields
```
However, @diraol has just suggested `self.oxm_match_fields = oxm_match_fields or OxmMatchFields()` which is seems to be a cleaner solution.